src/test/kotlin/uk/gov/justice/digital/hmpps/externalmovementsapi/integration/sync/SynTapApplicationIntTest.kt

```diff
@@ -90,7 +91,9 @@ class SynTapApplicationIntTest(
     contactPersonName: String? = null,
     applicationType: String = "SINGLE",
     audit: NomisAudit = NomisAuditGenerator.generate(),
+    id: UUID? = null,
```

Comment:
Would it be tricky to move this to the first param in line with the request object?